### ISSUE TITLE
Live stream URL not playing

### ISSUE THREAD

**Issue by codesalsa** [ampervue/ngJWPlayer#8]
Hi
My live stream url works fine with jwplayer in the website, but when I integrate it using ng-jWPlayer in my ionic project, its not working. Its playing normal .mp4 files but showing error for live stream url.

My live stream URL is - http://139.59.13.228:8081/live/stream/playlist.m3u8

Also tried with the rtmp url, still no success

rtmp://139.59.13.228:8080/live/stream

Can you help me in fixing this.

**Comment by dkarchmer** [comment]
Did you try to make it work on a non-ionic project first? This more likely than not an issue with the mobile app itself. Divide and conquer, so first try to get it to work on a small sample project running on your computer's browser. If that works, then the issue is related to ionic/cordova settings, and not an issue with this project.

**Comment by codesalsa** [comment]
I tried integrating it in a non-ionic basic angularjs project and still Live stream URL not working. mp4 videos playing well.

But 
http://139.59.13.228:8081/live/stream/playlist.m3u8 
url or 
rtmp://139.59.13.228:8080/live/stream
url, none of them working. Its showing "error loading Media"

**Comment by dkarchmer** [comment]
Are you sure you have the license that allows you to play them?

Try small example with regular javascript as JWPlayer recommends it. At the end of the day, this is just a directive,  which passes the options to the jwplayer library, so you should figure out how to use it outside Angular first, and then you should be able to just pass the same options.

Sorry, but you have not really shown there is a bug with the directive.

**Comment by codesalsa** [comment]
Yes, the license allowed to play, as I am using the same license to play on my website.

Using regular javascript it is already integrated my website here: http://kanaknews.com/live-streaming/

I am not telling its a bug in the directive, but need your help in solving the issue that I am facing.

I have attached my code for your reference.
[videoplay.zip](https://github.com/ampervue/ngJWPlayer/files/495305/videoplay.zip)

Also find attached the html file where it is already integrated and working fine

[indexx.zip](https://github.com/ampervue/ngJWPlayer/files/495309/indexx.zip)
